modules/host/contractmanager/sectorupdate.go

```diff
@@ -411,10 +417,15 @@ func (cm *ContractManager) AddSectorBatch(sectorRoots []crypto.Hash) error {
 
 	// Add each sector in a separate goroutine.
 	var wg sync.WaitGroup
+	semaphore := make(chan struct{}, maxParallelThreads)
```

Comment:
Can you add a quick comment here that says something like `// Ensure only 'maxParallelThreads' goroutines are running at a time`